### ISSUE TITLE
Fix Formatting Issues of the Newsletter

### ISSUE THREAD

**Issue by praneesha** [ballerina-platform/ballerina-dev-website#3281]
**Description:**
Need to review and fix the formatting issues of the newsletter. For example, see any newsletter in the list in [1].

[1] https://ballerina.io/community/newsletter/#past-issues

**Describe your task(s)**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by sahil-MK2** [comment]
i want to fix this issue

**Comment by praneesha** [comment]
@dinushan20 - We need to fix the top and bottom padding in all titles. Currently, there seems to be a lot of white space before and after the titles.

**Comment by dinushan20** [comment]
I was put that space to avoid feel packed between lines, If we reduced padding by 10px it looks like this 
![image](https://user-images.githubusercontent.com/73055030/170658684-b8e0bcfa-23a1-4e8d-845b-f63e93905d87.png)
This IMO not looks good since we have keep good spacing in all other places in the website @sm1990 WDYT?

**Comment by sm1990** [comment]
> I was put that space to avoid feel packed between lines, If we reduced padding by 10px it looks like this ![image](https://user-images.githubusercontent.com/73055030/170658684-b8e0bcfa-23a1-4e8d-845b-f63e93905d87.png) This IMO not looks good since we have keep good spacing in all other places in the website @sm1990 WDYT?

I agree with you @dinushan20. Without the padding it looks very packed.

**Comment by praneesha** [comment]
@dinushan20 @sm1990 - I am referring to the top and bottom padding of topics inside a particular newsletter. There seems to be a lot of white spaces as shown below.

For example, see https://ballerina.io/community/newsletter/2022-2/.

<img width="1272" alt="Screen Shot 2022-05-28 at 1 42 45 PM" src="https://user-images.githubusercontent.com/11707273/170816932-399d6bf8-8032-4985-904d-a3da4bc09135.png">

**Comment by dinushan20** [comment]
Ack @praneesha , Recent editions has this issue, I will fix it
